var/spack/repos/builtin/packages/duckdb/package.py

```diff
@@ -106,6 +107,8 @@ def setup_build_environment(self, env):
                 env.set(make_flag, "1")
             elif "~" + flag in self.spec:
                 env.set(make_flag, "0")
+        if self.spec.satisfies("@:0.10.2"):
+            env.set("OVERRIDE_GIT_DESCRIBE", "v{0}".format(self.spec.version))
```

Comment:
FYI. We encourage moving to Python f-strings (https://docs.python.org/3/tutorial/inputoutput.html) in packages as we transition to it in the core.

Reply:
Oh! Happily! Wasn't sure what the compatibility aim was.